nixos/tests/mysql.nix

```diff
@@ -1,24 +1,34 @@
-import ./make-test.nix ({ pkgs, ...} : {
-  name = "mysql";
-  meta = with pkgs.stdenv.lib.maintainers; {
-    maintainers = [ eelco chaoflow shlevy ];
+{ system ? builtins.currentSystem }:
+with import ../lib/testing.nix { inherit system; };
+let
+  packages = {
+    mysql = pkgs.mysql;
+    mariadb = pkgs.mariadb;
   };
 
-  nodes = {
-    master =
-      { pkgs, ... }:
+  make-mysql-test = package: makeTest {
+    name = "mysql";
+    meta = with pkgs.stdenv.lib.maintainers; {
+      maintainers = [ eelco chaoflow shlevy ];
+    };
 
-      {
-        services.mysql.enable = true;
-        services.mysql.initialDatabases = [ { name = "testdb"; schema = ./testdb.sql; } ];
-        services.mysql.package = pkgs.mysql;
-      };
-  };
+    nodes = {
+      master =
+        { pkgs, ... }:
+
+        {
+          services.mysql.enable = true;
+          services.mysql.initialDatabases = [ { name = "testdb"; schema = ./testdb.sql; } ];
+          services.mysql.package = pkgs.mysql;
```

Comment:
Are you sure this changes for the different tests?

Reply:
Ups, I changed it :)